dashboard/.streamlit/config.toml

```diff
@@ -0,0 +1,17 @@
+[logger]
```

Comment:
Should probably mention that this is the configuration file for the dashboard here

Reply:
This is a configuration file so I don't think comments are needed